REFERENCE.md

```diff
@@ -955,6 +955,12 @@ A queue emits also some useful events:
   // workers that crash or pause the event loop.
 })
 
+.on('lock-extension-failed', function (job) {
```

Comment:
I noticed that there is another argument, "err" that is missing in this documentation.